packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageMarketCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useDynamicLeverage } from '@deriv/api';
+import { WalletText } from '../../../../components/Base';
+import { DynamicLeverageTableColumnHeader } from './DynamicLeverageTableColumnHeader';
+import './DynamicLeverageMarketCard.scss';
+
+type TDynamicLeverageMarketCardProps = ReturnType<typeof useDynamicLeverage>['data'][0];
+
+export const DynamicLeverageMarketCard = ({ data, description, leverage, title }: TDynamicLeverageMarketCardProps) => (
```

Comment:
Right now we are using the following convention for all components:
`export const ComponentName: FC<TComponentProps> = (...)`

Reply:
Updated.